packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts

```diff
@@ -115,15 +115,6 @@ export async function _scaffoldLocalDebugSettings(
             EOL: os.EOL,
           }
         );
-
-        await fs.writeJSON(
```

Comment:
seems we only updated local debug's logic. is there any other logic that generates vscode settings?

Reply:
Nope, the generate vscode settings.json file only exist in local debug scaffold stage